lms/djangoapps/discussion/static/discussion/templates/fake-breadcrumbs.underscore

```diff
@@ -1,6 +1,8 @@
 <h6 class="hd-6 breadcrumbs">
     <span class="nav-item">
-        <a class="all-topics" href="">All Topics</a>
+        <a class="all-topics" href="">
+            <span class="fa fa-bars"></span>All Topics
```

Comment:
Good catch. According to the Pattern Library, it should be:

    <span class="icon fa fa-bars" aria-hidden="true"></span>

http://ux.edx.org/design_elements/icons/